odoo/addons/base/tests/test_translate.py

```diff
@@ -394,6 +394,24 @@ def test_107_duplicate_record_en(self):
         ])
         self.assertEqual(translation_fr.src, 'Customers', "Did not set English version as source")
 
+    # opw-2266864
+    def test_108_duplicate_record_fr_with_no_en_name(self):
+        self.env.ref('base.lang_fr').active = True
+        self.env['res.users'].with_context(active_test=False).search([]).write({'lang': 'fr_FR'})
+
+        langs = self.env['res.lang'].get_installed()
+        self.assertEqual([('en_US', 'English (US)'), ('fr_FR', 'French / Français')], langs, "Test did not started with expected languages")
+
+        category = self.env['res.partner.category'].with_context(lang='fr_FR').create({'name': 'French Name'})
+
+        name = category.with_context(lang=None).read(['name'])
+        self.assertEqual(name[0]['name'], "French Name", "Reference field not updated")
+
+        new_category = category.copy()
```

Comment:
maybe ensure that `new_category.name != category.name` so if we remove the override of `copy` at one point we need to adapt the test

Reply:
oh actually there is no override of `res.partner.category`. `res.groups` has a translatable name with an override if we want to test this behaviour